data/json/items/ammo.json

```diff
@@ -946,6 +946,7 @@
         "category" : "chems",
         "price" : 3500,
         "name" : "sprayable fungicide",
+        "name_plural" : "sprayable fungicide",
```

Comment:
Maybe I didn't say clearly, the `pantss` string appears in translator's working panel, but is never displayed in the game. The trailing 's' is automatically added to the translation template by `lang/extract_json_strings.py` script because **no explicit plural form is defined**. Then I fix this issue by declaring the plural form **explicitly**. Then the incorrect string `pantss` will no longer exist.